src/commands/inputs.rs

```diff
@@ -175,12 +182,19 @@ impl InputProcessor {
                     for item in m.items() {
                         let (k, v) = item.key_value();
                         let key_name: String = match k {
-                            Expr::Literal(LiteralExpr::String(k)) => {
-                                let text = k.text().expect("should have string text");
-                                let mut buffer = String::new();
-                                text.unescape_to(&mut buffer);
-                                buffer
-                            }
+                            Expr::Literal(LiteralExpr::String(k)) => k
```

Comment:
I made this change without actually taking a close look at the context (been a long time since I reviewed this bit of code). I'm not sure this is right, but it's late for me. I'll take another look in the morning, but this might need to be changed 🤔